packages/server/src/routes.ts

```diff
@@ -113,6 +114,10 @@ export const createRouter = (ctx: AppContext): express.Router => {
   // Update or create a DID doc
   router.post('/:did', async function (req, res) {
     const { did } = req.params
+    // TODO: Provide carve-out for big clients like Bluesky
+    await rateLimitPerHour(ctx, res, 'did-post-ip', req.ip, 1000)
```

Comment:
probably oughta process these all at the same time so we don't have to wait for 3 round trips (even tho redis is fast)

Reply:
Yeah, that's a good idea. Right now I have each rate limit split out by duration (each duration gets its own RateLimiter from the package), but I bet there's a way to combine either the Redis requests or combine the rate limit tracking per key (e.g. DID/IP)